examples/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM alpine
+EXPOSE 8080
```

Comment:
I think it should expose more ports

Reply:
What ports else do I need to export?

Reply:
When you run `all` it prints all used ports